### DISCUSSION TITLE
Python interpreter - return JSON created in Python script back to C++ application

### DISCUSSION THREAD

**Discussion by luz-arreola** [pybind/pybind11#4488] [Q&A]
Please bear with me...I've never used Python before.  I am currently able to execute the following script from within my C++:

**hello.py**

```
import json
x =  { "name":"John", "age":30, "city":"New York"}
return json

```
I want my C++ code to execute the Python script and for the script to return JSON.

This is what I currently have:

**C++ code:**

```
std::string script = readFile("./cl/stencils/hello.py");
pybind11::scoped_interpreter guard{};
pybind11::exec(script);

```
How do I obtain this json from within my C++ application?

Thank you!

**Comment by luz-arreola** [comment]
I was not able to obtain the json object returned by the python **script** but it is accessible to my C++ code using **pybind11::dict** which worked just fine.

**Comment by Dariusz1989** [comment]
you could also do json.dumps(myDict) and then return that as string & load using nlohmann::parse back in to json if ur json had mixt types/etc.
